zookeeper/source/zk_nn_client.cc

```diff
@@ -350,6 +350,29 @@ namespace zkclient{
         FileZNode znode_data;
         read_file_znode(znode_data, src);
         znode_data.under_construction = FILE_COMPLETE;
+		// set the file length
+		uint64_t file_length = 0;
+		auto file_blocks = std::vector<std::string>();
+		if (!zk->get_children(ZookeeperPath(src), file_blocks, error_code)) {
+			LOG(ERROR) << CLASS_NAME << "Failed getting children of " << ZookeeperPath(src) << " with error: " << error_code;
+			res.set_result(false);
```

Comment:
since there's no early return it looks like setting res to false here has no effect since it's put to true at the end of the function